tests/TestCase/Auth/DigestAuthenticateTest.php

```diff
@@ -515,7 +515,7 @@ protected function generateNonce($secret = null, $expires = 300, $time = null)
         $secret = $secret ?: Configure::read('Security.salt');
         $time = $time ?: microtime(true);
         $expiryTime = $time + $expires;
-        $signatureValue = md5($expiryTime . ':' . $secret);
+        $signatureValue = hash_hmac('sha1', $expiryTime . ':' . $secret, $secret);
```

Comment:
Is this just me, or shouldn't this be sha256?  seems to be causing some trouble for me.

Reply:
Are you suggesting sha256 for the stronger algorithm?

Reply:
no, due to inconsistency with the validNonce function

Reply:
I totally missed that :blush:. I'll get it fixed.

Reply:
great! Thanks. Had me pulling my hair out.